tests/README.md

```diff
@@ -0,0 +1,21 @@
+Running and creating tests                        {#running-and-creating-tests}
+==========================
+
+There are a number of tests included in RIOT. They are located in the
+[tests folder](https://github.com/RIOT-OS/RIOT/tree/master/tests). These tests
+allow basic functionality to be verified as well as provide an example of
+usage.
+
+
+Running automated tests
+-----------------------
+
+Some tests can be preformed automatically. The test automation scripts are
+defined in the `<test_application>/tests/` folder. They are written in python
+and interact with the test application code to do the validation. It is
```

Comment:
```suggestion
and interact through the uart with the test application code running on a board to do the validation. It is
```

Reply:
The suggestion is to explain the difference between the host python test and that there is a board running the test application.
The diff cannot be applied directly otherwise the line ending are wrong but cannot do better from here.